src/adera_app/features/jolt.cpp

```diff
@@ -623,7 +623,7 @@ static void rocket_thrust_force(BodyId const bodyId, ACtxJoltWorld const& rJolt,
 
     JPH::BodyID joltBodyId = BToJolt(bodyId);
     Quaternion const rot = QuatJoltToMagnum(bodyInterface.GetRotation(joltBodyId));
-    RVec3 joltCOM = bodyInterface.GetCenterOfMassPosition(joltBodyId) - bodyInterface.GetPosition(joltBodyId);
+    RVec3 joltCOM = bodyInterface.GetPosition(joltBodyId) - bodyInterface.GetCenterOfMassPosition(joltBodyId);
```

Comment:
For the pendulum effect the rockets are having?

The issue here is that GetCenterOfMassPosition takes (world) rotation in account when it shouldn't. I worked this out on https://github.com/Capital-Asterisk/osp-magnum/commit/463fe6e553fc5ffe2a4c192de964fcf13575836f , part of the commit I sent on the gdextension pull request. This can just be cherry-picked.

Reply:
Oh well perfect then.